recipe/recipe.yaml

```
@@ -16,7 +16,8 @@ source:
   sha256: 04b5d0bc5a1851ba1504d46bf9d7dbb355222a69f2cd440d54295db721b331f7
 
 build:
-  number: 1
+  number: 2
+  noarch: ${{ "python" if noarch else "" }}
```

is [minijinja's `none`](https://docs.rs/minijinja/latest/minijinja/tests/fn.is_none.html) the same as [`null` (the `noarch` default)](https://github.com/conda/ceps/blob/main/cep-0014.md#build-section)?

```suggestion
  noarch: ${{ "python" if noarch else none }}
```

---

Trying...

---

I think this works